exaudfclient/base/create_binary_wrapper.sh

```diff
@@ -5,6 +5,9 @@ CLIENT_BINARY=$1
 CLIENT_WRAPPER=$2
 WRAPPER_TEMPLATE=$3
 touch "$CLIENT_WRAPPER"
-cat "$WRAPPER_TEMPLATE" >> "$CLIENT_WRAPPER"
-echo >> "$CLIENT_WRAPPER"
-echo ./$(basename "$CLIENT_BINARY") '$*' >>  "$CLIENT_WRAPPER"
+
+{
+cat "$WRAPPER_TEMPLATE"
+echo
+echo ./"$(basename "$CLIENT_BINARY")" '$*'
+} >> "$CLIENT_WRAPPER"
```

Comment:
Reason:
" SC2129: Consider using { cmd1; cmd2; } >> file instead of individual redirects."

Reply:
This seems fine for me, actually much more elegant than before.